avocado/utils/cpu.py

```diff
@@ -114,3 +114,13 @@ def get_cpu_arch():
         return 'x86_64'
     else:
         return 'i386'
+
+def cpu_online_map():
+    """
+    Check out the available cpu online map
+    """
+    cpus = []
+    for line in open('/proc/cpuinfo', 'r').readlines():
```

Comment:
The use of `readlines()` is counterproductive as it creates list of all lines, while only `for line in open(...)` iterates the lines.